### DISCUSSION TITLE
General Topics

### DISCUSSION THREAD

**Discussion by sidney-eliot** [JoseConseco/GoB#361] [General]
For general stuff

**Comment by sidney-eliot** [comment]
Recently I've been using blender for more precise polygroup selection. And give the facemaps a random number as the name. Could you explain to me how ZBrush recognizes the numbers as colors, is there maybe some kind of pattern like all number between 0 - 500 are light blue for example or something like that. And when I'm already talking about this, would you also happen to know how many color variants ZBrush's polygroups can be

**Comment by kromar** [comment]
i did not understand that color translation of zbrush yet that is why im assigning random colors. the range goes from 0 to 65535 but i do not understand the pattern. I might give it another try some day.

**Comment by sidney-eliot** [comment]
I was going to ask something (text below) but found it out, I'll leave it here though, because I think this is quite a cool mechanic not many know about. The solution is to check unselected in the merge by distance pop up window, and that will do exactly what I described.

---

Not related to gob: Would you maybe know if there is a way of getting a merge by distance to not merge both verts it finds to the center of those 2 verts, but to merge close verts in such a way that verts are moved to the selected ones.

![image](https://user-images.githubusercontent.com/85735034/182199446-40585033-43be-42fc-8a5d-3db763737898.png)
And yes, in this example other methods off bridging would work, but for more complex things doing it my hand is a nightmare.

To me it just feels like every time I weld hardsurface stuff, it's a gamble if the surfaces are still smooth
